src/content-handlers/iiif/modules/uv-shared-module/css/mixins.less

```diff
@@ -16,12 +16,12 @@
 // 2. The use of `table` rather than `block` is only necessary if using
 //    `:before` to contain the top-margins of child elements.
 .clearfix() {
-  &:before,
-  &:after {
+  &::before,
```

Comment:
More of these single-colon to double-colon conversions. Are these semantically equivalent in LESS? (Sorry, this stuff isn't my strength; just wanted to double-check!)